web/src/main/java/org/fao/geonet/monitor/onlineresource/OnlineResourceCheckerWps.java

```diff
@@ -27,14 +27,14 @@ public void setOnlineResource(String uuid, final Element onlineResource) {
     }
 
     @Override
-    public boolean check() {
+    public CheckResult check() {
         long start = System.currentTimeMillis();
         try {
             String time = getTime();
 
             // If the wfs request fails or returns csv without time value
             if (time == null)
-                return false;
+                return new CheckResult(false, "No time value passed.");
```

Comment:
This logic needs to be revisited when we relook at the WPS monitor for AWS/WPS - we don't have to have a time value anymore.  Perhaps add a todo.  In any case, "No time stamp found for layer" would be a better error message.

Reply:
No worries.  Changing message.